packages/ui-tamagui/src/__tests__/index.test.tsx

```diff
@@ -40,21 +36,32 @@ it('should render a Tamagui component', () => {
 
   expect(element.asFragment()).toMatchInlineSnapshot(`
     <DocumentFragment>
-      <view
-        style="flex-direction: row;"
+      <span
+        class="t_light is_inversed _dsp_contents"
       >
-        <text
-          accessibilityrole="header"
-          style="color: rgb(23, 23, 23); text-transform: none; font-family: InterBold; letter-spacing: -1.5px; font-size: 44px; line-height: 53; margin: 0px 0px 0px 0px;"
-        >
-          Hello
-        </text>
-        <text
-          style="color: rgb(23, 23, 23); font-family: Inter; letter-spacing: 0; font-size: 14px; line-height: 23;"
+        <span
+          class=" t_light _dsp_contents is_Theme"
         >
-          Welcome to Tamagui!
-        </text>
-      </view>
+          <div
+            class="_dsp-flex _ai-stretch _fd-row _fb-auto _bxs-border-box _pos-relative _mih-0px _miw-0px _fs-0"
+            data-testid="root"
+          >
+            <h1
+              class="is_H1 font_heading _col-675002279 _tt-1440318557 _ff-299667014 _fow-1366436877 _ls-905095908 _fos-1477259397 _lh-1677663454 _dsp-inline _bxs-border-box _ww-break-word _whiteSpace-normal _mt-0px _mr-0px _mb-0px _ml-0px _ussel-auto"
```

Comment:
I think the snapshot was easier to review when it was printing out the react native definition.

With all these obscure class names its hard to see what is happening. Also maybe it is possible the class names change for some reason but the styles still stay the same?

Reply:
Yeah, needs further DD for sure. React Native one was a bit easier, but not really working. To make it work, we would need to switch to a Native Testing Library.

I think we can remove class="" from the snapshot to reduce boilerplate. I will look into it in a follow-up if you don't mind!